#! /bin/sh
patch -p1 -l -f $* < $0
exit $?

From df5a89e6929479dc1bf301974fe7d607d024bc1f Mon Sep 17 00:00:00 2001
From: Andre Heinecke <aheinecke@gnupg.org>
Date: Thu, 12 Nov 2020 16:07:14 +0100
Subject: [PATCH] qt: Fix export of qDebug GpgME::Error overload

* lang/qt/src/debug.cpp: Include debug.h to get the export decl.
---
 lang/qt/src/debug.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/lang/qt/src/debug.cpp b/lang/qt/src/debug.cpp
index e4974ad9..b563ac0c 100644
--- a/lang/qt/src/debug.cpp
+++ b/lang/qt/src/debug.cpp
@@ -36,6 +36,7 @@
 #endif

 #include "error.h"
+#include "debug.h"

 #include <QDebug>

--
2.20.1
